### PR TITLE
Adds a new minor gas reaction.

### PR DESCRIPTION
**Pull request by as334**
## About The Pull Request

Adds a new small reaction for some of the endgame gases. You can combine hot Stimulum and Pluoxium(with some nitryl to catalyze and plasma as fuel), and it will release a high power radioactive energy ball like the ones released by fusion. The angle of the shot is also dependent on the concentrations of water vapor and nitryl within the reaction, so you can modify those amounts to aim which way the ball is shot for those elaborate atmos snipes. Also significantly lowers the heat requirements of Nitryl, as it was way too high and now there's more motivation to obtain it.

Deletes the fusion test canisters that I forgot to take out of the code earlier.

## Why It's Good For The Game

Atmos is still lacking in reactions that produce meaningful effects in the world, this is an example of a fun, small one that's pretty versatile and adds slightly more incentive to acquiring late game gases. The reaction itself is very flexible, and allows a couple cool possibilities. 
Possible things you can try with this:
- Atmosia Gatling Gun
- Using it with a TTV for a portable weapon
- Have it going in distro to have random energy blasts shoot around the station

## Changelog
:cl:
add: New Gas Reaction, Stimulum Energy Balls. Combine 100 degree Pluoxium, Stimulum and small amounts of Plasma and Nitryl to shoot off a deadly radioactive energy ball.
add: The reaction will consume the plasma, pluoxium and stimulum and release small amounts of heat. The presence of water vapor will change the direction in which it shoots. See if you can figure out how to aim it.
tweak: Nitryl now forms at 22,000 degrees, a significantly lower temperature.
fix: fixed a few things
del: Removed some old left-behind fusion test canisters.
/:cl:

### CODE REVIEW

**Comment by blueybloke** [comment]
+rep, would be nice to be able to have more things to do with late game gases.

**Comment by plapatin** [comment]
atmos techs are getting an awful amount of love lately aren't they

**Comment by as334** [comment]
Every job deserves more love, I'm just trying to do my part.

**Comment by SouDescolado** [comment]
> Every job deserves more love, I'm just trying to do my part.

Even Assistants?

**Comment by duncathan** [comment]
this is fun :+1:

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 1]
```diff
@@ -459,3 +459,42 @@
...
+	location.fire_nuclear_particle(ball_shot_angle)
+	cached_gases[/datum/gas/carbon_dioxide][MOLES] += 4*pluox_used
+	cached_gases[/datum/gas/nitrogen][MOLES] += 8*stim_used
+	cached_gases[/datum/gas/pluoxium][MOLES] -= pluox_used
+	cached_gases[/datum/gas/stimulum][MOLES] -= stim_used
+	cached_gases[/datum/gas/plasma][MOLES] *= 0.5 //Consumes half the plasma each time.
+	if(energy_released)
+		var/new_heat_capacity = air.heat_capacity()
+		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
+			air.temperature = (air.temperature*old_heat_capacity + energy_released)/new_heat_capacity
```
Comment (AnturK) [inline code/modules/atmospherics/gasmixtures/reactions.dm]: Needs a clamp

**Review by AnturK — CHANGES_REQUESTED** [review]
Looks fine outside of this, if bit abusable.

**Comment by as334** [comment]
Sorry this took so long to update, I switched to a new computer recently and it took me a while to get around to getting my code stuff set up.

**Pull request merged**